src/components/guidance-content/info/InformationItems.js

```diff
@@ -0,0 +1,617 @@
+    <p>Summary listing of all datasets selected by dataset filter criteria.</p>
+    <p>
+      Each row of the table represents a group of up to 3 datasets.
+      We group datasets by Model Run, Start Date and End Date,
+      and each such group is labelled accordingly as shown in the
+      "Label in selectors" column.
+    </p>
+    <p>
+      The "Yearly", "Seasonal", and "Monthly" columns indicate whether a
+      dataset with that timescale (averaging period) is available in the
+      group.
+    </p>
+  </LabelWithInfo>
+);
+
+///////////////////////////////
+// Dataset selectors
+///////////////////////////////
+
+export const datasetSelectorLabel = (
+  <LabelWithInfo label='Dataset'>
+    <p>
+      Select a single dataset to display from all of those that match the Model,
+      Emissions Scenario, and Variable selected.
+    </p>
+    <p>
+      The map and the graph(s) have independently selected datasets.
+      That is, they may be displaying different datasets simultaneously.
+    </p>
+    <p>
+      On the map, use the {mapSettingsControl} button to access the Dataset selector.
+      The selected dataset is shown in the legend in the lower left of the map.
+    </p>
+    <p>
+      On the graph(s), the Dataset selector appears directly above the graph.
+    </p>
+    <p>
+      Datasets are identified by a combination of
+      model run id (e.g., <code>r1i1p1</code>) and
+      averaging period (e.g., <code>1961-1990</code>).
+    </p>
+  </LabelWithInfo>
+);
+
+export const timeOfYearSelectorLabel = (
+  <LabelWithInfo label='Time of Year'>
+    <p>
+      Select the portion of the year over which data values are averaged
+      before those portion averages are averaged over a multi-decadal period.
+    </p>
+  </LabelWithInfo>
+);
+
+///////////////////////////////
+// Export/download controls
+///////////////////////////////
+
+const downloadFormats = (<span>
+  Data may be downloaded in Microsoft Excel compatible format (XSLX)
+  or in Comma Separated Values (CSV) format.
+  For details on these formats, see <Link to='/help/general'>Help</Link>.
+</span>);
+
+const exportDataLabel = 'Export Data';
+
+export const downloadGraphDataLabel = (
+  <LabelWithInfo label={exportDataLabel}>
+    <p>
+      Click a button to download the selected data to your computer.
+    </p>
+    <p>
+      The data downloaded is that shown on the graph.
+      For details on the layout and content of the exported data,
+      see <Link to='/help/general'>Help</Link>.
+    </p>
+    <p>{downloadFormats}</p>
+  </LabelWithInfo>
+);
+
+export const xslxButtonLabel = 'XSLX';
+export const csvButtonLabel = 'CSV';
+
+
+///////////////////////////////
+// Map
+///////////////////////////////
+
+const LeafletControlContainer = ({ children }) => (
+  <span className='leaflet-touch'>
+      <span className='leaflet-control-container'>
+        {children}
+      </span>
+    </span>
+);
+
+const mapSettingsControl = (
+  <span>
```

Comment:
This doesn't seem to be used anywhere; I'm guessing it'll make more sense in a later PR and is just an artifact of having to split up the big PR.  I'm not entirely sure about the reasoning for including a control in the file-of-help-text, but I imagine it'll be clearer later.

Reply:
Or, wait, is it just meant to be a representation of the Map Settings button, not the actual button, and its only use is in other help elements? In that case, could it get a more descriptive name, like mapSettingsControlRepresentation, or ... uh ... something shorter than that.

Reply:
Okay, I see that it's used in help elements in the massive uni-branch, and was just a victim of the Great Untangling. Never mind!